### PR TITLE
Add: Support for station variable 6B

### PR DESCRIPTION
**Pull request by glx22**
https://github.com/OpenTTD/OpenTTD/pull/10953 added station variable 6B to get extended station id of nearby tiles.

Since roadstop feature exists only in OpenTTD 14+, I just replaced var68 with var6B.

~~But for stations an extra compatibility layer is required.~~ OpenTTD 14 is out now, so just replace var68 with var6B.

### CODE REVIEW

**Comment by WenSimEHRP** [comment]
For station grfs that has station tiles that relies on ids this PR would be useful. Examples are https://github.com/WenSimEHRP/jppluswins and https://github.com/openttd-china-set/china-set-stations

**Comment by PeterN** [comment]
Is the duplicate key warning an issue?

File: nml/actions/action2var_variables.py [hunk 1]
```diff
@@ -353,13 +369,14 @@ def platform_info_fix_var(var, info):
...
     'nearby_tile_station_id'        : {'var': 0x68, 'start':  0, 'size':  8, 'param_function': signed_tile_offset},
-    'nearby_tile_original_gfx'      : {'var': 0x68, 'start':  8, 'size':  2, 'param_function': signed_tile_offset, 'value_function': value_equals(0)},
-    'nearby_tile_same_grf'          : {'var': 0x68, 'start':  8, 'size':  2, 'param_function': signed_tile_offset, 'value_function': value_equals(1)},
-    'nearby_tile_other_grf'         : {'var': 0x68, 'start':  8, 'size':  2, 'param_function': signed_tile_offset, 'value_function': value_equals(2)},
+    'nearby_tile_same_grf'          : {'var': 0x68, 'start':  8, 'size':  2, 'param_function': signed_tile_offset, 'value_function': value_equals(0)},
+    'nearby_tile_other_grf'         : {'var': 0x68, 'start':  8, 'size':  2, 'param_function': signed_tile_offset, 'value_function': value_equals(1)},
+    'nearby_tile_original_gfx'      : {'var': 0x68, 'start':  8, 'size':  2, 'param_function': signed_tile_offset, 'value_function': value_equals(2)},
```
Comment (PeterN) [inline nml/actions/action2var_variables.py:374]: Looks like you reverted this

**Comment by glx22** [comment]
I failed when solving a conflict.

**Review by PeterN — APPROVED** [review]

**Pull request merged**